### DISCUSSION TITLE
Warp permission negation issue.

### DISCUSSION THREAD

**Discussion by NobleStrike** [EssentialsX/Essentials#4477] [Ideas and Feature Suggestions]
So I am trying to limit certain warps for certain ranks, the players already have hundreds of warps they enjoy freely with staff adding and removing warps all the time. So they would need the perm essentials.warps.* and then negated perms of all the warps like essentials.warps.zone1 under the same rank. Issue is the essentials.warps.* overlaps all the other warp perms. Requiring either a higher ladder rank to work with the negations ,which doesn't mix well with a default rank, OR all warp perms added by hand then the few you want to negate, which isn't really doable when you have over 200 warps being changed by without the owners awareness all the time.

What I'm asking for is a way to just negate some warp perms without all the jerry rigging the bulk of the rest of the warps if possible. 
Such as if a perm is listed as negated to override/ignore the config setting "per-warp-permission: false" prevent warping to that perm with a # note saying "Negated warp perms bypass this setting!" or add another config line to allow this to more cleanly.

Thank you for your time.

**Comment by benwoo1110** [comment]
What permission plugin do you use? Setting `essentials.warps.zone1` to false should override wildcard of `essentials.warps.* ` if you using luckperms.

**Reply by NobleStrike** [reply]
Using Pex for main server since 2013 and Luckyperms for the rest of the network since 2019. It is probably just a Pex bug in how its read.
